plugin/config.json

```diff
@@ -53,6 +53,12 @@
       "description": "Log level - info, warn, error or debug",
       "value": "info",
       "Settable": [ "value"]
+	},
+	{
+      "name": "VMCI_VERSION",
+      "description": "Request VMCI version",
+      "value": "",
```

Comment:
The client MUST have the version defined in the code. Please do not rely on the env variable for this